### PR TITLE
Origin/feat/refactoring

### PR DESCRIPTION
**Pull request by 0xkf**
combined 2 directories (node and browser) to 1 directory(browser_node)
When you hit "npm run node" , it will work on node environment.
When you hit "npm run browser " , it will work on browser environment.

I edited that setting type module and compiling typescript code to javascript for node environment.

I would like to hear your feedback.
Thank you for reading

### CODE REVIEW

**Review by mhchia — COMMENTED** [review]
Hey @0xkf, thanks for your excellent work! I think it can be merged with one more thing: can you remove `examples/browser` and `examples/node`? Also, can you put everything under `examples/browser_node/` to `examples/` and remove `example/browser_node/`?

**Comment by 0xkf** [comment]
> Hey @0xkf, thanks for your excellent work! I think it can be merged with one more thing: can you remove `examples/browser` and `examples/node`? Also, can you put everything under `examples/browser_node/` to `examples/` and remove `example/browser_node/`?

Thank you for your rebiew!
OK!
I will do that soon!

**Comment by 0xkf** [comment]
I changed as you said.
Can you review??
@mhchia

**Review by mhchia — COMMENTED** [review]
Directory changes look good! It came to my mind that READMEs should also be updated. Could you do them altogether in this PR?

For `examples/README.md`, I've made a [draft](https://hackmd.io/@mhchia/ByAyORJe6) for it and you can use it directly.

For `README.md`, the Example section can be changed from
```md
See [example for both NodeJS](./examples/node/) and [example for browser](./examples/browser/).
```
to
```md
See the [example](./examples) that can be run in both NodeJS and browser.
```

Let me know too if you have an idea of what we should write for them

**Comment by 0xkf** [comment]
I have updated README, watching your draft.
Could you review??

**Comment by 0xkf** [comment]
@mhchia

File: examples/README.md [hunk 1]
```diff
@@ -20,19 +20,18 @@ In a new terminal, run:
 $ npx hardhat node
 ```
 
-3. Run the web server
+3. Run the example in a browser
+(take note that you should hit "npm run clean" after hitting "npm run browser")
```
Comment (mhchia) [inline examples/README.md:24]: Is this note still relevant? If so, I'm thinking we can rephrase it to 'Try `npm run clean` if you've made changes to the code but the web page is not updated'

**Review by mhchia — APPROVED** [review]
Sorry, I missed that we should also remove *.js like `examples/src/configs.js` and `examples/src/index.js`. I've also added one comment about your note in the readme. Should be ready after these

**Comment by 0xkf** [comment]
I have updated as you said.
・deleted js files and realized npm run node experience without that js files.
・updated Readme as you said
@mhchia

**Comment by mhchia** [comment]
Merged. Thanks for your contribution
